docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
     <label>organization-defined personnel or roles</label>
-  </param>
-  <param id="ac-1_prm_2">
-    <label>organization-defined frequency</label>
-  </param>
-  <param id="ac-1_prm_3">
-    <label>organization-defined frequency</label>
-  </param>
-  <prop class="label">AC-1</prop>
-  <part class="statement" id="ac-1_smt">
-    <p>The organization:</p>
-    <part class="item" id="ac-1_smt.a">
-      <prop class="label">a.</prop>
-      <p>Develops, documents, and disseminates to <insert param-id="ac-1_prm_1"/>:</p>
-      <part class="item" id="ac-1_smt.a.1">
-        <prop class="label">1.</prop>
-        <p>An access control policy that addresses purpose, scope, roles,
-           responsibilities, management commitment, coordination among
-           organizational entities, and compliance; and</p>
+</param>
+<param id="ac-01_odp.01">
+    <prop name="label" class="sp800-53a" value="AC-01_ODP[01]"/>
+    <label>personnel or roles</label>
+    <guideline>
+        <p>personnel or roles to whom the access control policy is to be
+        disseminated is/are defined;</p>
+    </guideline>
+</param>
+<param id="ac-01_odp.02">
+    <prop name="label" class="sp800-53a" value="AC-01_ODP[02]"/>
+    <label>personnel or roles</label>
+    <guideline>
+        <p>personnel or roles to whom the access control procedures are
+        to be disseminated is/are defined;</p>
+    </guideline>
+</param>
+<param id="ac-01_odp.03">
+    <prop name="alt-identifier" value="ac-1_prm_2"/>
+    <prop name="label" class="sp800-53a" value="AC-01_ODP[03]"/>
+    <select how-many="one-or-more">
+        <choice>organization-level</choice>
+        <choice>mission/business process-level</choice>
+        <choice>system-level</choice>
+    </select>
+</param>
+[... params ac-01_odp.04 through ac-01_odp.08 ...]
+{{< / highlight >}}
+
+`<param>` elements define parameter values for the control that permit (and may require) setting in context. Typically, a catalog will expose parameters where profiling applications are expected either to define appropriate values themselves (that is, values appropriate to a baseline) or to permit setting at higher levels of implementation (such as system plans implementing a baseline). These values are available for reference (and even display) in the language of the control as presented, using corresponding `<insert>` elements, as described in detail below.
+
+OSCAL parameters are worth special coverage as a topic in themselves. As shown here, they can and commonly do have internal structure and properties of their own, sometimes specific to a particular application, requirement or use case. For example, in this data set, `<prop>` elements are given to provide extra information -- `alt-identifier` values -- helping assure parameters' correspondences across versions of SP 800-53. Consumers of OSCAL content can determine where and how such information is useful to them, and ignore or discard it when it constitutes noise.
```

Comment:
Minor, non-blocking: should we just open up an issue to document parameters with special coverage on its own page? I agree that it is a worthy topic, but definitely want OSCAL leads to confirm that and agree if it's a priority or not. 😅